hack/ci/windows.ps1

```diff
@@ -774,11 +786,36 @@ Try {
     # Run the unit tests inside a container unless SKIP_UNIT_TESTS is defined
     if (($null -eq $env:LCOW_MODE) -and ($null -eq $env:LCOW_BASIC_MODE)) {
         if ($null -eq $env:SKIP_UNIT_TESTS) {
+            $ContainerNameForUnitTests = $COMMITHASH + "_UnitTests"
             Write-Host -ForegroundColor Cyan "INFO: Running unit tests at $(Get-Date)..."
             $ErrorActionPreference = "SilentlyContinue"
-            $Duration=$(Measure-Command {docker run -e DOCKER_GITCOMMIT=$COMMITHASH$CommitUnsupported docker hack\make.ps1 -TestUnit | Out-Host })
+            $Duration=$(Measure-Command {docker run --name $ContainerNameForUnitTests -e DOCKER_GITCOMMIT=$COMMITHASH$CommitUnsupported docker hack\make.ps1 -TestUnit | Out-Host })
+            $TestRunExitCode = $LastExitCode
             $ErrorActionPreference = "Stop"
+
+            # Saving for artifacts......
+            $unitTestsContPath="$ContainerNameForUnitTests`:c`:\gopath\src\github.com\docker\docker\bundles"
+            $JunitExpectedContFilePath = "$unitTestsContPath\junit-report-unit-tests.xml"
+            docker cp $JunitExpectedContFilePath "$TEMPORIG"
+            if (-not($LastExitCode -eq 0)) {
+                Throw "ERROR: Failed to docker cp the unit tests report ($JunitExpectedContFilePath) to $TEMPORIG"
+            }
+
+            if (Test-Path "$TEMPORIG\junit-report-unit-tests.xml") {
+                Write-Host -ForegroundColor Magenta "INFO: Unit tests results($TEMPORIG\junit-report-unit-tests.xml) exist. pwd=$pwd"
+            } else {
+                Write-Host -ForegroundColor Magenta "ERROR: Unit tests results($TEMPORIG\junit-report-unit-tests.xml) do not exist. pwd=$pwd"
+            }
+            
+            # Saving where jenkins will take a look at.....
+            $bundlesDir = "bundles"
+            if (-not (Test-Path $bundlesDir)) { New-Item $bundlesDir -ItemType Directory | Out-Null }
```

Comment:
oh; this one could also use `-Force` (equivalent of `mkdir -p`);

```powershell
 New-Item -Force "bundles" -ItemType Directory | Out-Null
```

Reply:
Done - however instead of explicit use of the value of the directory name, I am choosing to keep the variable because it is used elsewhere too.